drizzlepac/pixtosky.py

```diff
@@ -4,101 +4,109 @@
 
     :License: :doc:`/LICENSE`
 
-    PARAMETERS
+"""
+
+import warnings
+import numpy as np
+
+from . import util
+from . import wcs_functions
+from stwcs import wcsutil
+from . import __version__
+from astropy.utils.decorators import deprecated_renamed_argument
+
+__taskname__ = 'pixtosky'
```

Comment:
Is `__taskname__` still necessary? Was it something required by teal?

Reply:
I'm not sure. I was trying to minimize changes as there are already many (sorry), but I'm guessing this is a relic of TEAL. I will follow up after with another PR removing all old TEAL things.

Reply:
I did fix this in [bb304c9dc2249be45a44a554aa5b9ad0ebfa77ec](https://github.com/spacetelescope/drizzlepac/pull/2079/commits/bb304c9dc2249be45a44a554aa5b9ad0ebfa77ec)